site/source/docs/introducing_emscripten/community.rst

```diff
@@ -12,10 +12,13 @@ Welcome to the Emscripten community!
 Get in touch
 ============
 
-The best ways to contact us for support are the *mailing list* and the *IRC* channel. Ask questions, share your ideas, or just join the conversation.
+The best ways contact the community are:
 
+- The GitHub `Issue Tracker <https://github.com/emscripten-core/emscripten/issues?state=open>`_
 - Mailing list: `emscripten-discuss <http://groups.google.com/group/emscripten-discuss>`_
-- IRC: **#emscripten** on `irc.mozilla.org <https://wiki.mozilla.org/IRC>`_
+- Real-time chat: `#emscripten on Discord <https://discord.gg/zvAMHvs>`_ (there was also an IRC channel on Mozilla IRC, but they are shutting those servers down).
```

Comment:
The invite link is expired

Reply:
A fix is in the works in #10532, should land soon. Can use the link in there for now, sorry about any annoyance.